src/pages/index.jsx

```diff
@@ -50,15 +56,19 @@ export const query = graphql`
 
 export default function IndexPage({
   data: {
+    homeIntro: { nodes: homeIntro },
     allApproaches: { nodes: allApproaches },
     firstCases: { nodes: firstCases },
     firstMembers: { nodes: firstMembers },
     firstPosts: { nodes: firstPosts },
   },
 }) {
+    console.log(homeIntro);
   return (
     <Layout>
       <SEO title="Home" />
+      <h3>Foursevens</h3>
```

Comment:
No need for this to be an H3. Just `div` is probably fine.

Reply:
i made it a div as you requested but i saw it in our website it was h3 so i did it the same